model/http/registry.yaml

```diff
@@ -27,7 +27,15 @@ groups:
           The attribute value MUST consist of either multiple header values as an array of strings
           or a single-item array containing a possibly comma-concatenated string, depending on the way
           the HTTP library provides access to headers.
-        examples: ['http.request.header.content-type=["application/json"]', 'http.request.header.x-forwarded-for=["1.2.3.4", "1.2.3.5"]']
+
+          Examples:
+
+          - A header `Content-Type: application/json` should be recorded as the `http.request.header.content-type`
+              attribute with value `["application/json"]`.
+          - A header `X-Forwarded-For: 1.2.3.4, 1.2.3.5` should be recorded as the `http.request.header.x-forwarded-for`
+              attribute with value `["1.2.3.4", "1.2.3.5"]` or `["1.2.3.4, 1.2.3.5"]` depending on the HTTP library.
+
+        examples: ['["application/json"]', '["1.2.3.4", "1.2.3.5"]']  # TODO: make it array of arrays
```

Comment:
> TODO: make it array of arrays

is this not supported by tooling today?

Reply:
unfortunately no, tooling expects it to be a string array and complains. @jerbly is fixing it in the https://github.com/open-telemetry/weaver/pull/707